### PR TITLE
replaced travis CI with GHA GI

### PR DESCRIPTION
**Pull request by johnricords**

### CODE REVIEW

**Comment by johnricords** [comment]
tested locally, results:

```
make terraform/pytest PYTEST_ARGS="-v --nomock"
[bootstrap]: Using external tardigrade-ci, skipping auto-init
[pytest//Volumes/Development/tardigrade-ci/tests/terraform_pytest] Starting Python tests found under the directory "/Volumes/Development/tardigrade-ci/tests/terraform_pytest"
pytest /Volumes/Development/tardigrade-ci/tests/terraform_pytest -v --nomock 
================================================ test session starts ================================================
platform darwin -- Python 3.10.9, pytest-7.2.1, pluggy-1.0.0 -- /Users/johnricords/.local/opt/python@3.10/bin/python3.10
cachedir: .pytest_cache
rootdir: /Volumes/Development
collected 4 items                                                                                                   

../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[schedule_based] PASSED        [ 25%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[premade_policy] PASSED        [ 50%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[event_and_schedule] PASSED    [ 75%]
../tardigrade-ci/tests/terraform_pytest/test_terraform_install.py::test_modules[event_based] PASSED           [100%]

=========================================== 4 passed in 97.46s (0:01:37) ============================================
[pytest//Volumes/Development/tardigrade-ci/tests/terraform_pytest]: Tests executed!
```

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+on:
+  pull_request:
+
+concurrency:
+  group: lint-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    uses: plus3it/actions-workflows/.github/workflows/lint.yml@41056300ba78d8be98b45f057aea246f8c4f30cf
```
Comment (lorengordon) [inline .github/workflows/lint.yml:11]: This is the old SHA, for plus3it/actions-workflows v1.0.0. Please update all these to the SHA for v1.0.1

**Review by lorengordon — CHANGES_REQUESTED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (johnricords) [reply .github/workflows/lint.yml]: corrected

**Review by lorengordon — APPROVED** [review]
🚀

**Pull request merged**